spec/support/account_reset_helper.rb

```diff
@@ -1,8 +1,15 @@
 module AccountResetHelper
   def create_account_reset_request_for(user)
-    AccountReset::CreateRequest.new(user).call
-    account_reset_request = AccountResetRequest.find_by(user_id: user.id)
-    account_reset_request.request_token
+    request = AccountResetRequest.find_or_create_by(user: user)
```

Comment:
What was the rationale for removing the service call and duplicating the code?

Reply:
Because the service call sends an SMS and caused webmock to complain that an external call to Twilio was being made. This particular test shouldn't have to depend on mocking Twilio calls. All we want in this helper is to simulate a request for the user, not have the service perform everything else.